### ISSUE TITLE
Currency Symbol like rupees euro not working

### ISSUE THREAD

**Issue by aneesh2usman** [xhtml2pdf/xhtml2pdf#558]
Currency Symbol like **rupees** **euro** not working. i am used **UTF-8** the code write below

pisa.CreatePDF(html.encode("UTF-8"), file_object , encoding='UTF-8',
                   link_callback=link_callback)

**Comment by skwars7** [comment]
I was able to use rupee currency after importing ttf file into HTML

**Comment by luisza** [comment]
Yes,  you need to include a external font that support ruppee currency.
Sorry, now system fonts are limited.
